etc/mersenne.c

```diff
@@ -68,10 +68,10 @@ static int is_mersenne(long s, int *pp)
    return res;
 }
 
-/* square root of a long < 65536 */
-static long i_sqrt(long x)
+/* square root of a int < 65536 */
```

Comment:
this comment is now lying for some architectures...

why even the change from `long` to `int`?

Reply:
I'll revert this part.